swift/apple/FirezoneNetworkExtension/Adapter.swift

```diff
@@ -35,34 +35,41 @@ enum AdapterError: Error {
 }
 
 // Loosely inspired from WireGuardAdapter from WireGuardKit
-class Adapter {
-  typealias StartTunnelCompletionHandler = ((AdapterError?) -> Void)
+class Adapter: @unchecked Sendable {
 
-  private var callbackHandler: CallbackHandler
+  /// Command sender for sending commands to the session
+  private var commandSender: Sender<SessionCommand>?
 
-  private var session: WrappedSession?
-
-  // Our local copy of the accountSlug
-  private var accountSlug: String
-
-  /// Network settings
-  private var networkSettings: NetworkSettings?
+  /// Event receiver for receiving events from the session
+  private var eventReceiver: Receiver<Event>?
 
   /// Packet tunnel provider.
   private weak var packetTunnelProvider: PacketTunnelProvider?
 
   /// Network routes monitor.
   private var networkMonitor: NWPathMonitor?
 
-  /// Used to avoid path update callback cycles on iOS
```

Comment:
I am surprised to see this gone!

Reply:
it's failing due to not finished moving parts (conditional compilation 🙈 ) . should be fixed in a jiffy!

Reply:
> I am surprised to see this gone!

I believe it should have been removed with these changes: https://github.com/firezone/firezone/pull/10022/files#diff-a84e8f62a17ac67f781019e6ac0456567fd18ffa7c13b3248609d78debb6480eL342

we now prevent cycles on iOS this way https://github.com/firezone/firezone/commit/290b0c298f7d3345d3376f6b20c362825298019f ?

Reply:
Yes! This is correct - good catch.